rfhub2/db/model/keyword.py

```diff
@@ -24,3 +25,36 @@ def __repr__(self):  # pragma: no cover
     @classmethod
     def default_ordering(cls) -> List[OrderingItem]:
         return [OrderingItem("name")]
+
+    @staticmethod
+    def create(data: model.KeywordCreate) -> "Keyword":
+        return Keyword(
+            name=data.name,
+            doc=data.doc,
+            args=data.args,
```

Comment:
@MaciejWiczk you could add here conversion of `tags: List[str]` from `data` to `tags: str` in DB model by using json serialization

Reply:
json.loads/json.dumps will be enough for now